# Radiative Transfer and Stars

### ALMA observed a face-on protoplanetary disk which is in a star forming region 100 pc away from us. The measured flux is 100 mJy at 1mm. Assuming the disk temperature is 20 K, and the radio emission from the disk is due to thermal radiation, what is the total dust mass in this disk? We also assume that the disk is optically thin along our line of sight and the opacity is due to 1 cm dust's geometrical absorption .   (1 Jy=1e-23 erg/s/cm$^2$/Hz)


In [5]:
# cgs unit, length unit is cm, mass unit is gram, time unit is second
import numpy as np

sigmacont=5.67051e-5
acont=7.5657e-15
ev=1.602e-12
hcont=6.625e-27
ccont=3.e10
kcont=1.381e-16
rcont=8.314472e7
me=9.109e-28
Gcont=6.67e-8
Msun=1.989e33
Mearth=5.972e27
Mjupiter=1.898e30
Lsun=3.827e33
rsun=6.96e10
rjupiter=7.149e9
rearth=6.371e8
au=1.49598e13
pc=3.085678e18
mh=1.67352e-24
yr=365.*24.*3600
pi=3.1415926

In [2]:
# Step 1, what is the wavelength where the disk's spectrum peaks ? What is the simplified 
# relationship between the intensity and disk temperature.

From __Wien's law__:
$$
\lambda_{max}T = 0.29 \quad\textrm{cm K} \\
\lambda_{max} = \dfrac{0.29}{T} \quad\textrm{cm K}
$$

In [3]:
T = 20 # temp in K
lmax = 0.29/T
print('λₘₐₓ =', lmax, 'cm')

λₘₐₓ = 0.014499999999999999 cm


Simplified relationship between intensity and disk (blackbody) temperature:

\begin{align}
I = B = \dfrac{\sigma T^4}{\pi}
\tag{A}
\end{align}

In [4]:
# Step 2, When the emission is optically thin, what is the relationship between 
# the intensity and the optical depth? 

\begin{align}
I_\nu = S_\nu\tau_\nu,
\tag{B}
\end{align}

where $\tau_\nu$ is the _optical depth_, and the _source function_ is

\begin{align}
S_\nu = B_\nu
\tag{C}
\end{align}

for objects in _local thermal equilibrium_ (LTE).

In [1]:
# Step 3, What is the opacity of 1 cm dust grains?

If $d = 1$ cm, then

\begin{align}
\kappa &= 1/d \cdot \textrm{cm}^3/\textrm{g} \\
&= 1 \quad\textrm{cm}^2/\textrm{g}
\tag{D}
\end{align}


In [ ]:
# Step 4, what is the relationship between flux received by the telescope and the intensity 

\begin{align}
F = I\dfrac{A}{d^2},
\tag{E}
\end{align}

where $A$ is the cross-sectional area of the disk (face-on), and $d$ is the distance to the disk.

In [6]:
# Step 5, Use flux and opacity to calculate dust mass=rho*A*L

The total dust mass in the disk is

\begin{align}
M = \rho A L,
\tag{F}
\end{align}

where $\rho$, $A$, and $L$ respectively are the density, area, and thickness of the disk.  If the integrated optical depth is $\alpha L$, where the absorption coefficient $\alpha = \rho \kappa$, then the optical depth is
$$
\tau = \rho \kappa L.
$$

Solving for $\rho$, we find

\begin{align}
\rho = \dfrac{\tau}{\kappa L}.
\tag{G}
\end{align}

Substituting Eq. A into E,

\begin{align}
F = \dfrac{\sigma T^4 A}{\pi d^2},
\tag{H}
\end{align}

and solving for $A$, we find

\begin{align}
A = \dfrac{\pi d^2 F}{\sigma T^4}.
\tag{I}
\end{align}

Substituting Eqs. G and I into F,

\begin{align}
M = \left( \dfrac{\tau}{\kappa L} \right) \left( \dfrac{\pi d^2 F}{\sigma T^4} \right) L.
\tag{J}
\end{align}

But, substituting Eq. C into B, and solving for $\tau$, we have

\begin{align}
I &= B\tau \\
\tau &= \dfrac{I}{B} \\
&= \dfrac{B}{B} \\
&= 1
\tag{K}
\end{align}

where in the second to last step we substituted $I = B$ from Eq. A.  Finally, substituting our results from Step 3 (Eq. D) and Eq. K into J, and canceling the $L$ terms, we arrive at

\begin{align}
M = \dfrac{\pi d^2 F}{\sigma T^4}.
\tag{J}
\end{align}

Using the values provided, we can calculate the total dust mass in the disk $M$ as follows:

In [8]:
Jy = 1e-23    # jansky or erg/s/cm^2/Hz
d = 100*pc    # distance to disk in parsecs
T = 20        # temp in kelvin
F = 100e-3*Jy # measured flux in mJy
M = pi*d**2*F/sigmacont/T**4
print('M =', M, 'g')

M = 3.296924259825697e+16 g


Thus, the total dust mass in the disk is $3.3 \times 10^{16} \textrm{ g}$ or $3.3 \times 10^{13} \textrm{ kg}$.

# Homework due Next Monday, Chapter 2, question 1